js/components/graph/__tests__/Sidebar.test.js

```diff
@@ -34,6 +34,17 @@ describe("Sidebar", () => {
     expect(container.getByText("FCE Count: 0.5")).toBeDefined()
   })
 
+  it("Click the course code on the sidebar should decrease the FCE count", async () => {
```

Comment:
This is a good description, just make sure to follow the same grammatical form as the other descriptions. In this case use "Clicking" instead of "Click"